biz.aQute.repository/src/aQute/bnd/repository/maven/provider/MavenBndRepository.java

```diff
@@ -305,6 +317,12 @@ private Jar getSources(Tool tool, Processor context, String path) throws Excepti
 	private Jar getJavadoc(Tool tool, Processor context, String path, Map<String, String> options, boolean exports)
 		throws Exception {
 		Jar jar = toJar(context, path);
+		if (path != null && jar == null) {
+			logger.warn(
+				"A javadoc path was set in the -maven-release instuction, but no javadoc could be found in {} (base path: {}) ",
+				path, context.getBaseURI()
```

Comment:
I think the 2nd arg should just be `options.context.getBase()`. I don't see why we would want to use a URI here. We also should not `toString()` it in case logging does not happen. Leave the `toString()` call to the logging code.